.travis.yml

```diff
@@ -11,10 +11,14 @@ before_install:
 install:
     - eval "$(gimme 1.11.4)"
 script:
-    -   cd go-controller;
-    -    make;
-    -    make windows;
-    -    make gofmt;
-    -    make install.tools;
-    -    make lint;
-    -    make check;
+    - pushd go-controller
+    -    make
+    -    make windows
+    -    make gofmt
+    -    make install.tools
+    -    make lint
+    -    make check
+    - popd
+    - pushd dist/images
+    -    [[ -n "$(git diff --stat origin/master.. | grep dist/images/Dockerfile)" ]]; make all
```

Comment:
I assume you mean `&&` rather than `;` here, but why not keep the `if`?

Note that I was not objecting to anything in the previous version, I was merely commenting that I didn't know what that syntax meant. (Unless it was supposed to actually just be a shell script with a `-` before each line, in which case, yes, I was objecting.)

Reply:
@danwinship I don't know that syntax either, so I simplified stuff. Fixed the ; -> &&